src/components/experiment-tracking/runs-list/runs-list.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import RunsList from '.';
+import { ApolloProvider } from '@apollo/client';
+import { client } from '../../../apollo/config';
+import { shallow } from 'enzyme';
+import { setup } from '../../../utils/state.mock';
+
+const runDataList = [
+  {
+    bookmark: false,
+    id: 'run1',
+    timestamp: new Date('October 15, 2021 03:24:00').toISOString(),
+    title: 'Run 1',
+    notes: 'notes of run 1 is a duck',
+  },
+  {
+    bookmark: false,
+    id: 'run2',
+    timestamp: new Date('October 15, 2021 03:26:00').toISOString(),
+    title: 'Run 2',
+    notes: 'notes of run 2 is a fish',
+  },
+  {
+    bookmark: false,
```

Comment:
Maybe we should also test when a bookmark is `true` and then we get the two-accordions view?

Reply:
indeed - on that note, I have observed that there are currently no tests set up for the run-list for the bookmark journey - it would be great if we can set some tests for that. I will raise a separate ticket for that. 

Your suggested test condition would sound like a perfect test case for the bookmark journey test setup to be dealt with in that ticket - what do you think?

Reply:
Yeah we could and probably should be robust and create a separate ticket tailored more towards bookmarks. We'll still need a test for the distinct accordions, and I think that could live in this file, as those are part of the runs list.

You can do it so quick too, because if you use `bookmark: true` on one of the runs then the condition on line 38 becomes this, and then that basically checks for two accordions:

```js
expect(wrapper.find('.runs-list__wrapper').length).toBe(2);
```